crates/ruff_cli/src/commands/clean.rs

```diff
@@ -8,11 +9,26 @@ use walkdir::WalkDir;
 
 use ruff::fs;
 use ruff::logging::LogLevel;
-use ruff_cache::CACHE_DIR_NAME;
+use ruff_cache::{cache_dir, CACHE_DIR_NAME};
 
 /// Clear any caches in the current directory or any subdirectories.
-pub(crate) fn clean(level: LogLevel) -> Result<()> {
+pub(crate) fn clean(cache_dir_overwrite: Option<PathBuf>, level: LogLevel) -> Result<()> {
     let mut stderr = BufWriter::new(io::stderr().lock());
+    let pwd = std::env::current_dir()?;
+    let path = cache_dir(cache_dir_overwrite, &pwd);
+    if level >= LogLevel::Default {
+        writeln!(
+            stderr,
+            "Removing cache at: {}",
+            fs::relativize_path(&path).bold()
+        )?;
+    }
+    remove_dir_all(path)?;
```

Comment:
do we have any safeguard that we don't [accidentally remove user data](https://www.theregister.com/2015/01/17/scary_code_of_the_week_steam_cleans_linux_pcs/), e.g. checking for `CACHEDIR.TAG`?